sematic/ui/src/components/RunTime.tsx

```diff
@@ -2,22 +2,30 @@ import { Typography } from "@mui/material";
 import { Run } from "../Models";
 
 export function RunTime(props: { run: Run; prefix?: string }) {
-  const { run, prefix } = props;
+  const { run, prefix = "" } = props;
   let startedAt = new Date(run.started_at || run.created_at);
   let endedAt = new Date();
   let endTimeString = run.failed_at || run.resolved_at;
   if (endTimeString) {
     endedAt = new Date(endTimeString);
   }
 
-  let durationMS: number = endedAt.getTime() - startedAt.getTime();
+  let durationS: number = Math.round(
+    (endedAt.getTime() - startedAt.getTime()) / 1000
+  );
+  let displayH: number = Math.floor(durationS / 3600);
+  let displayM: number = Math.floor((durationS - displayH * 3600) / 60);
+  let displayS: number = Math.round(
+    durationS - displayH * 3600 - displayM * 60
+  );
 
   return (
     <Typography fontSize="small" color="GrayText">
-      {prefix}
-      {Number.parseFloat((durationMS / 1000).toString()).toFixed(0)} seconds
-      {/*on&nbsp;
-              {new Date(run.created_at).toLocaleString()*/}
+      {prefix + " "}
+      {displayH > 0 ? displayH.toString() + "h " : ""}
+      {displayM > 0 ? displayM.toString() + "m " : ""}
+      {displayS > 0 ? displayS.toString() + "s" : ""}
+      {durationS == 0 ? "<1s" : ""}
```

Comment:
Until now we would show the total number of seconds. Now we show the h/m/s breakdown.